verde/tests/test_spline.py

```diff
@@ -99,6 +100,25 @@ def test_spline():
     )
 
 
+def test_spline_cv_scoring():
+    "Check scoring parameter works with SplineCV"
+    region = (100, 500, -800, -700)
+    synth = CheckerBoard(region=region)
+    data = synth.scatter(size=1500, random_state=1)
+    coords = (data.easting, data.northing)
+    # Compare SplineCV to results from Spline with cross_val_score
+    for score in ["r2", "neg_root_mean_squared_error"]:
+        spline = Spline()
```

Comment:
```suggestion
        spline = Spline(damping=None, mindist=1e-5)
```

Best to set the same parameters here explicitly to avoid breaking in case the default ever changes.